dev/tests/unit/Magento/FunctionalTestFramework/Test/Util/ActionMergeUtilTest.php

```diff
@@ -288,6 +316,26 @@ public function testInvalidSecretFunctions()
         $actionMergeUtil->resolveActionSteps($actionObject);
     }
 
+    /**
+     * Test Exception Handler for merging actions
+     *
+     * @throws \Exception
+     */
+    public function testMergeActionsException()
```

Comment:
We had two test classes for a single class. I combined them and copied this test over from the incorrect class.